herald/src/main/java/io/heraldprox/herald/sensor/datatype/PseudoDeviceAddress.java

```diff
@@ -103,6 +107,7 @@ public PseudoDeviceAddress(final long value) {
         this.address = decode(data);
     }
 
+    @Nullable
```

Comment:
Result is never null. How about updating `PseudoDeviceAddress(final byte[] data)` to `PseudoDeviceAddress(@NonNull final byte[] data)`?